### PR TITLE
Refactored code for improved clarity

### PR DESCRIPTION
**Pull request by David-Ademola**
# Changes made to the code
- Moved all the import statements to the start of the script.
- Removed unused namespaces.
- Made changes to some comments to improve understanding of succeeding code block.
- Added more extensions to the ```extensions_to_skip``` list.

Overall, these changes aim to improve clarity of the code for future users.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -22,9 +23,6 @@
     # timeout=120,
 )
 def generate_response(system_prompt, user_prompt, *args):
-    import openai
-    import tiktoken
```
Comment (swyxio) [inline main.py:26]: can you undo this? it is intentional for modal https://modal.com/docs/guide/custom-container#importing-python-packages

Comment (swyxio) [reply main.py:26]: maybe add a comment for future readers

**Review by swyxio — CHANGES_REQUESTED** [review]
appreacite the PR! minor changes please

File: main.py (hunk 1 above)
Comment (David-Ademola) [reply main.py:26]: Already on it. Thanks for your feedback.

**Pull request closed without merging**